src/qt/guiutil.cpp

```diff
@@ -708,7 +707,7 @@ QString formatServicesStr(quint64 mask)
     }
 
     if (strList.size())
-        return strList.join(" & ");
+        return strList.join(" ");
```

Comment:
> I agree with @jonatack and @Talkless

I have an impression that @jonatack' [suggestion](https://github.com/bitcoin-core/gui/pull/293#issuecomment-826357677) is:
> I'd suggest the comma-space separator as it requires fewer characters and is a well-understood separator.